js/web/script/test-runner-cli-args.ts

```diff
@@ -0,0 +1,392 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import minimist from 'minimist';
+import npmlog from 'npmlog';
+import {InferenceSession} from 'onnxruntime-common';
+
+import {WebGLFlags} from '../lib/backend-onnxjs';
+import {WebAssemblyFlags} from '../lib/backend-wasm';
+import {Logger} from '../lib/onnxjs/instrument';
+import {Test} from '../test/test-types';
+
+/* eslint-disable max-len */
+const HELP_MESSAGE = `
+test-runner-cli
+
+Run ONNX.js tests, models, benchmarks in different environments.
+
+Usage:
+ test-runner-cli <mode> ... [options]
+
+Modes:
+ suite0                        Run all unittests, all operator tests and node model tests that described in white list
+ suite1                        Run all unittests, all operator tests and all model tests that described in white list
+ model                         Run a single model test
```

Comment:
We don't support model test directly at onnxruntime. Put a comment about how to run a model test.